backend/src/database/repositories/memberRepository.ts

```diff
@@ -1775,6 +1795,247 @@ where m."deletedAt" is null
     }))
   }
 
+  static async mergeSuggestionsByUsername(
+    numberOfHours,
+    options: IRepositoryOptions,
+  ): Promise<IMemberMergeSuggestion[]> {
+    const transaction = SequelizeRepository.getTransaction(options)
+
+    const seq = SequelizeRepository.getSequelize(options)
+
+    const tenant = SequelizeRepository.getCurrentTenant(options)
+
+    const query = `
+    -- Define a CTE named "new_members" to get members created in the last 2 hours with a specific tenantId
+    WITH new_members AS (
+      SELECT id, "tenantId"
+      FROM members
+      WHERE "createdAt" >= now() - INTERVAL :numberOfHours
```

Comment:
I think to pass `:numberOfHours` you will need to construct an interval like so:

```sql
WHERE "createdAt" >= now() - ( :numberOfHours || ' hours')::interval
```